make/common/NativeCompilation.gmk

```diff
@@ -974,6 +974,10 @@ define SetupNativeCompilationBody
               $(CD) $$($1_SYMBOLS_DIR) && \
                   $$($1_OBJCOPY) --add-gnu-debuglink=$$($1_DEBUGINFO_FILES) $$($1_TARGET)
 
+        else ifeq ($(call isTargetOs, aix), true)
+          $1_DEBUGINFO_FILES := $$($1_SYMBOLS_DIR)/$$($1_NOSUFFIX).debuginfo
+          $1_CREATE_DEBUGINFO_CMDS := $(CP) $$($1_TARGET) $$($1_DEBUGINFO_FILES)
```

Comment:
This looks like you are just copying the linked binary and calling it .debuginfo. Is the idea that the original binary then gets stripped and this is just saving the unstripped binary? I don't know how things are done in AIX, but this seems like a rather strange way of achieving external symbol information to me.

Reply:
You're essentially correct Eric, on AIX there is no such tool as OBJCOPY on Linux platforms, so you cannot physically remove the symbolic information from the native compiled object. So this CREATE_DEBUG_INFO_CMDS copies the object containing symbols, prior to the STRIP_CMD being executed as part of the make of $$($1_TARGET) recipe:
```
                  $$($1_CREATE_DEBUGINFO_CMDS)
                  $$($1_STRIP_CMD)
```
The external debuginfo is then passed by the user of AIX dbx using -B <.debuginfo>

You make a good point though that this looks confusing, so I will update the PR with a comment to this rule.
thanks

Reply:
I've added a comment in the make file in the refresh

Reply:
Thanks for the clarification and the comment!